submissions/korotushak96/memory-pair-game/js/script.js

```diff
@@ -0,0 +1,111 @@
+const gameInit = ()=>{
+    const scene = document.querySelector('.scene');
+    const pokemons = [
+        {
+            id: 0,
+            src: 'img/bulbasaur.png'
+        },
+        {
+            id: 1,
+            src: 'img/charmander.png'
+        },
+        {
+            id: 2,
+            src: 'img/eevee.png'
+        },
+        {
+            id: 3,
+            src: 'img/lapras.png'
+        },
+        {
+            id: 4,
+            src: 'img/pikachu.png'
+        },
+        {
+            id: 5,
+            src: 'img/squirtle.png'
+        }
+    ];
+    
+    let contentArray = [...pokemons,...pokemons];
+    contentArray.sort(function() { return 0.5 - Math.random() });
+
+    let pokeHTML = '';
+    contentArray.forEach(({id, src}, index)=>{
+        pokeHTML+=`
+            <div class="card is-flipped " data-id="${id}" data-index='${index}'>
+                <div class="card__face card__face--front">
+                    <img src='${src}'>
+                </div>
+                <div class="card__face card__face--back ">
+                    <img src='img/back.jpg'>
+                </div>
+            </div>
+        `
+    });
+    
+    scene.insertAdjacentHTML('afterbegin', pokeHTML);
+    let firstCard = -1;
+    let secondCard = -1;
+    let disabed = 0;
+    const cards = [...document.querySelectorAll('.card')];
+    const backSide = document.querySelectorAll('.card__face--back')
```

Comment:
`const CARDS ... const BACK_SIDE` Read this when you have time [const](https://javascript.info/variables#constants)

Reply:
I've never understood this strange convention. I think, it had sence before ES6 (when was only var).
But I'm too young to argue with bearded coders ;)